documentation/docs/guides/sessions/conversing-with-goose.md

```diff
@@ -0,0 +1,282 @@
+---
+sidebar_position: 12
+title: Conversing with Goose
+sidebar_label: Conversing with Goose
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { PanelLeft, Paperclip, Edit2, ScrollText, Send, GripVertical, X, ChevronUp, ChevronDown } from 'lucide-react';
+
+Goose provides features you can use to share information and communicate during sessions. Learn to refine your conversations and manage context more effectively.
+
+## Edit Message
+
+Edit your previously sent messages to refine conversations and correct course. This is useful when:
```

Comment:
I'm thinking we should move your explanation of editing a message out of the tab area and into the description area

```suggestion
Edit your previously sent messages to refine conversations and correct course.  

Editing any message in the session gives you complete control over the conversation history by overwriting all the context that follows the edited message. Your change can be as simple as fixing a path in your last message or completely starting over from a given point. 

This is useful when:
```